app/mesh/changelog.md

```diff
@@ -4,6 +4,28 @@ no_search: true
 no_version: true
 ---
 
+## 1.2.6
+
+> Released on 2021/05/13
+
+### Changes
+
+Built on top of [Kuma 1.1.6](https://github.com/kumahq/kuma/blob/master/CHANGELOG.md#116).
+
+- Intermediate Certificate Authorities (CAs) are now supported with Vault integration.
+- You can now specify any and all tags in a Traffic Permission policy for Vault integration.
+- You can now specify TCP and HTTP health checks at the same time in the same policy. The health check policy also 
+now includes a `reuse_connection` option.
+- The `--gateway` flag is now available in the CLI.
```

Comment:
Is this a new flag, or is it just newly added to the CLI? If it's new, would be nice to have a few word describing what it does.

Reply:
not new, new to do it with the CLI